assets/scripts/im.pontoWikiaAppsBridge.js

```diff
@@ -0,0 +1,41 @@
+define('im.pontoWikiaAppsBridge', ['ponto', 'im.window'], function (ponto, w) {
+	'use strict';
+
+	var pontoWikiaAppClass = 'Linker',
+		poiArticleLinkClass = 'poi-article-link';
+
+	/**
+	 * @desc sends link URL to native wikia app
+	 * @param {Element} link
+	 */
+	function sendLinkUrl(link) {
+		ponto.invoke(pontoWikiaAppClass, 'goTo', {
+			url: link.href,
+			title: link.dataset.articleTitle.replace(/ /g, '_')
```

Comment:
I would add some check here. We don't want to throw exceptions when there is no `link.dataset.articleTitle`